tests/pom.xml

```diff
@@ -169,6 +169,15 @@
             <artifactId>apicurio-registry-utils-converter</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>postgresql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>kafka</artifactId>
+            <version>1.12.4</version>
```

Comment:
Let's pull all versions up to the properties section.  Either of the root pom.xml or this one if it's really a test-only dep.

Reply:
fixed